src/components/ListItem.js

```diff
@@ -0,0 +1,44 @@
+import { AiFillEdit } from "react-icons/ai";
+
+export default function ListItem({
+  index,
+  name,
+  isComplete,
+  onTogglePresence,
+  onDelete,
+  handleRenamingToDo,
+}) {
+  function handleDeleteClick() {
+    onDelete(index);
+  }
+
+  function handleToggleClick() {
+    onTogglePresence(name, index);
+  }
+
+  function handleEditName(index) {
+    const editMessage = prompt("Please rename here...");
+    handleRenamingToDo(editMessage, index);
+  }
+  const listCompleted = isComplete ? "ListItemCompleted" : "ListItemPending";
```

Comment:
Nice usage of the ternary operator!
The class names though are not being consistent with what I saw in the rest of the project. Something you should take a look :)